web_sdk/BUILD.gn

```diff
@@ -528,7 +520,16 @@ if (!is_fuchsia) {
       ]
     }
 
-    sources += [ "$root_out_dir/flutter_web_sdk/flutter.js" ]
+    # flutter.js full sources
+    foreach(source, flutter_js_source_list) {
+      sources += [ "$root_out_dir/flutter_web_sdk/flutter_js/$source" ]
+    }
+
+    # flutter.js bundled source and sourcemap
+    sources += [
+      "$root_out_dir/flutter_web_sdk/flutter_js/flutter.js",
+      "$root_out_dir/flutter_web_sdk/flutter_js/flutter.js.map",
```

Comment:
We should publish both optimized and raw `flutter.js` so that it is possible to troubleshoot issues in debug and profile builds. In fact, until the size of `flutter.js` is more than a blip, I wouldn't bother minifying, just strip the TypeScript type annotations. Alternatively, if we could roll `esbuild` all the way to the Flutter SDK (how big is the binary?), and then provide optimization/minification options in the Flutter Tool. This way they engine build of `flutter.js` can be distributed in original source.

Reply:
We actually are publishing both with this PR now! We ship the minified version, the original source, and the sourcemaps that map between them.

Reply:
Good question! about how big it is without minification?

Reply:
Right now uncompressed it's 14kb before minification, 4kb after, so not terribly significant. The main reason to be bundling is to merge multiple files into one so that we don't have a complex waterfall with multiple roundtrips. Obviously right now there is just a single file, but I'm trying to prepare to modularize it into separate .ts files. Since we have sourcemaps and are shipping the original source, I don't really see a downside to actually minifying it since we're already bundling though.